spec/lib/nomis_client/court_hearings_spec.rb

```diff
@@ -2,31 +2,71 @@
 
 require 'rails_helper'
 
-RSpec.describe NomisClient::CourtHearing do
+RSpec.describe NomisClient::CourtHearing, with_nomis_client_authentication: true do
   describe '.post' do
     subject(:court_hearing_post) {
       described_class.post(booking_id: booking_id, court_case_id: court_case_id, body_params: {})
     }
 
     let(:booking_id) { 1111 }
     let(:court_case_id) { 2222 }
+    let(:response_status) { 201 }
+    let(:response_body) { '{}' }
```

Comment:
I think we don't need these lines

Reply:
I added them so we're explicit/have fewer things to know about outside of the test. You are correct, though

Reply:
Oh, I wanted the change the response status to what we expect for the create, actually.